core/hostInfo/os_c.go

```diff
@@ -49,8 +49,12 @@ func (o OSCollector) GetLabels(prefix string) (map[string]string, error) {
 		return map[string]string{}, errors.Wrap(err, constants.OSGetDataError+"failed to get OS data")
 	}
 	labels := map[string]string{
-		fmt.Sprintf("%s.%s", prefix, "docker_version"):       o.getDockerVersion(o.InfoData, false)["dockerVersion"],
-		fmt.Sprintf("%s.%s", prefix, "linux_kernel_version"): utils.SemverTrunk(osData["kernelVersion"], 2),
+		fmt.Sprintf("%s.%s", prefix, "docker_version"):                 o.getDockerVersion(o.InfoData, false)["dockerVersion"],
+		fmt.Sprintf("%s.%s%s", prefix, getOSName(), "_kernel_version"): utils.SemverTrunk(osData["kernelVersion"], 2),
+		fmt.Sprintf("%s.%s", prefix, "os"):                             getOSName(),
+	}
+	if getOSName() == "windows" {
+		labels[fmt.Sprintf("%s.healthcheck.skip", prefix)] = "true"
```

Comment:
So my higher-level issue is that this is a very specific bandaid to a much more general problem of defining what "capabilities" a host has, what ones the container requires, and then only scheduling on to the intersection of them.  We're going to end up with a proliferation of bandaid labels.

Ignoring that, that is better label.
  - There should be a default value documented which preserves the current behavior.
  - I would probably make it singular
  - And flip the last 2 segments so they group by service

So in summary... I vote

`io.rancher.infra_service.healthcheck.deploy={never|always}`